src/test/java/com/amazon/dlic/auth/http/jwt/HTTPJwtAuthenticatorTest.java

```diff
@@ -21,26 +35,12 @@
 import java.util.Date;
 import java.util.HashMap;
 import java.util.Map;
-import javax.crypto.SecretKey;
-
-import com.google.common.io.BaseEncoding;
-import org.apache.hc.core5.http.HttpHeaders;
-import org.junit.Assert;
-import org.junit.Test;
-
-import org.opensearch.OpenSearchSecurityException;
-import org.opensearch.common.settings.Settings;
-import org.opensearch.security.user.AuthCredentials;
-import org.opensearch.security.util.FakeRestRequest;
-
-import io.jsonwebtoken.JwtBuilder;
-import io.jsonwebtoken.Jwts;
-import io.jsonwebtoken.SignatureAlgorithm;
-import io.jsonwebtoken.security.Keys;
 
 import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.fail;
 
+;
```

Comment:
Looks like this leaked in. Please remove.

Reply:
Thanks, It fixed and force updated